app/services/adapters/amazon_ses_adapter.rb

```diff
@@ -19,7 +19,7 @@ def self.send_mail( opts = {} )
     private
 
     def self.client
-      Aws::SES::Client.new
+      Aws::SES::Client.new(region: 'eu-west-1')
```

Comment:
Ouch.

Is it worth adding a comment explaining why this is still pointing at Ireland?

Reply:
done